gradle/libs.versions.toml

```diff
@@ -194,9 +194,9 @@ sqldelightGradlePlugin = { module = "app.cash.sqldelight:gradle-plugin", version
 sqldelightJdbcDriver = { module = "app.cash.sqldelight:jdbc-driver", version = "2.0.0" }
 sqldelightMysqlDialect = { module = "app.cash.sqldelight:mysql-dialect", version = "2.0.0" }
 sqldelightRuntime = { module = "app.cash.sqldelight:runtime", version = "2.0.0" }
-tink = { module = "com.google.crypto.tink:tink", version = "1.6.1" }
-tinkAwskms = { module = "com.google.crypto.tink:tink-awskms", version = "1.6.1" }
-tinkGcpkms = { module = "com.google.crypto.tink:tink-gcpkms", version = "1.6.1" }
+tink = { module = "com.google.crypto.tink:tink", version = "1.13.0" }
+tinkAwskms = { module = "com.google.crypto.tink:tink-awskms", version = "1.9.1" }
```

Comment:
Should this match the following version at 1.10.0?

Reply:
Great question, when I looked up the packages I noticed they'd diverged so went with the latest stable versions which didn't introduce a dependency conflict (which in this case was in-fact latest):
- https://mvnrepository.com/artifact/com.google.crypto.tink/tink-awskms
- https://mvnrepository.com/artifact/com.google.crypto.tink/tink